onnx/helper.py

```diff
@@ -334,6 +334,99 @@ def float32_to_bfloat16(fval: float, truncate: bool = False) -> int:
     return (ival + rounded) >> 16
 
 
+def float32_to_floate4m3(fval: float, scale: float = 1.0) -> int:
+    """
+    Convert a float32 value to a floate4m3 (as int).
+
+    :param fval: float to convert
+    :param scale: scale
```

Comment:
Add some description for scale and also explaination for default value 1.0 basically when not specified fp32 will be truncated to fp8.

Reply:
The function documentation now refers to a page in the main documentation.